src/go/mongolib/proto/system.profile.go

```diff
@@ -13,8 +13,6 @@ type SystemProfile struct {
 	AllUsers        []interface{} `bson:"allUsers"`
 	Client          string        `bson:"client"`
 	CursorExhausted bool          `bson:"cursorExhausted"`
-	DocsExamined    int           `bson:"docsExamined"`
-	NscannedObjects int           `bson:"nscannedObjects"`
```

Comment:
@igroene NscannedObjects removed (should be deprecated) and DocsExamined used instead.